src/interface/ContextInterface.tsx

```diff
@@ -137,7 +147,6 @@ export async function retrieveVocabularyData(): Promise<boolean> {
     "OPTIONAL {?contextIRI dcterms:title ?title. }",
     "graph ?contextIRI {",
     "?vocab a <https://slovník.gov.cz/datový/pracovní-prostor/pojem/slovníkový-kontext> ",
```

Comment:
Changed this back to the previous setting; therefore no overall changes made here.